resource_customizations/numaplane.numaproj.io/ISBServiceRollout/actions/discovery.lua

```diff
@@ -0,0 +1,13 @@
+local actions = {}
+actions["enable-full-promote"] = {["disabled"] = true}
+actions["disable-full-promote"] = {["disabled"] = true}
```

Comment:
Consider setting a custom display name and icon for the actions: https://github.com/argoproj/argo-cd/pull/22202

Reply:
Is there an icon that Argo Rollouts is using that we should consider using as well?

Reply:
If not, we may want to do the icon as an enhancement later instead of right now.

Reply:
There is not, the icon feature is relatively new.